actions/install_provisioning_profile.rb

```diff
@@ -1,19 +1,20 @@
 module Fastlane
   module Actions
     module SharedValues
-      INSTALL_PROVISIONING_PROFILE_CUSTOM_VALUE = :INSTALL_PROVISIONING_PROFILE_CUSTOM_VALUE
     end
 
     class InstallProvisioningProfileAction < Action
       def self.run(params)
-        if ENV['TAB_PROVISIONING_PROFILE_PATH'] != nil
-          provisioning_profile_path=ENV['TAB_PROVISIONING_PROFILE_PATH']
-          provisioning_profile_uuid = `grep UUID -A1 -a #{provisioning_profile_path} | grep -io \"[-A-Z0-9]\\{36\\}\"`
-          provisioning_profile_destination = "#{ENV['HOME']}/Library/MobileDevice/Provisioning\\\ Profiles/#{provisioning_profile_uuid.strip}.mobileprovision"
-          `cp #{provisioning_profile_path} #{provisioning_profile_destination}`
-          UI.success("Installed profile at path #{params[:provisioning_profile_path]} succesfully")
-        else 
-          UI.message("Skipping installing provisioning profile since TAB_PROVISIONING_PROFILE_PATH is not defined")
+        profile_paths = Dir.glob("./**/*.mobileprovision")
+        if profile_paths.empty?
+          UI.message("Skipping installing provisioning profiles since no profiles were found.")
+        else
+          profile_paths.each do |path|
+            uuid = `grep UUID -A1 -a #{path} | grep -io \"[-A-Z0-9]\\{36\\}\"`
+            destination = "#{ENV['HOME']}/Library/MobileDevice/Provisioning\\\ Profiles/#{uuid.strip}.mobileprovision"
+            `cp #{path} #{destination}`
```

Comment:
It seems wild to me that we need that ugly `\\\` in the middle of that dir name 🤔 . Might be able to avoid it if we do `FileUtils.cp path destination` instead of shelling out to `cp` ?

Reply:
💪 f-yeah. Much better. Also catches the error now.